section-09-differential-testing/model-package/mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
```

Comment:
this mypy file is intimidating. Are we going to explain what all these params are for?

Reply:
We touched on it in earlier sections. If we wanted to talk about it more, section 9 is not the place